File: server/Models/Subscriber.cs
namespace theJourney.Models;
public class Subscriber
{
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string Email { get; set; }
    public string CreatorId { get; set; }
    public Account Creator { get; set; }
}

File: client/src/models/Subscriber.js
export class Subscriber{
    constructor(data){
        this.id = data.id
        this.email = data.email
        this.creator = data.creator
        this.creatorId = data.creatorId
        this.createdAt = new Date(data.createdAt).toLocaleTimeString()
        this.updatedAt = new Date(data.updatedAt).toLocaleTimeString()
    }
}